### DISCUSSION TITLE
Question about client notify

### DISCUSSION THREAD

**Discussion by tyctor** [NilCoalescing/djangochannelsrestframework#184] [FAQ]
Hi

first thank you for this module.

I use `@model_observer` to notify clients about model changes
the message is sent to all clients,
is there some way to avoid sending the message to the client which made that change?
because that client has already received same data in the CRUD response

thanks for any hint

**Comment by johnthagen** [comment]
When I do this with bare `django-channels`, I generate a UUID in the `connect()` method and save it to the instance's state (e.g. `self.id`). That way the session ID can be compared when sending responses and filtered to avoid looping back to the same client.

**Comment by hishnash** [comment]
Due to how the observer works with django signals there is no easy way to suppress this, I suppose it might be possible to set a UUID with the db action thread then read this within the signal when the value is encoded before being sent over the channel layer thus letting you include the consumer id in the message. So you can filter it in the consumer when the message arrives.
